content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -123,6 +124,14 @@ your cluster. Those fields are:
   See [Label Selectors](/docs/concepts/overview/working-with-objects/labels/#label-selectors)
   for more details.
 
+- **matchLabelKeys**  is a set of pod label keys to select the pods over which spreading will be calculated. The keys are used to lookup values from the pod labels, those key-value labels are ANDed with labelSelector to select the group of existing pods over which spreading will be calculated for the incoming pod. Keys that don't exist in the pod labels will be ignored. A null or empty list means only match against labelSelector.
```

Comment:
- please wrap this, similar to the rest of the file.
- I'd write `labelSelector` in code format (inside backticks).

Reply:
Also, please outline how this field is useful to an end user. It's good to document the mechanism but we're not showing why any of this might be worth doing.

Reply:
Done